device-types/Digi/Digi-Passport-48.yml

```diff
@@ -0,0 +1,116 @@
+---
+manufacturer: Digi
+model: Digi Passport 48
+slug: digi-passport-48
+part_number: '70002264'
+u_height: 1
+is_full_depth: false
```

Comment:
1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. This appears to be listed in the datasheet (6.48 lbs or 2.96 kg)